apps/ensrainbow/src/cli.ts

```diff
@@ -112,6 +117,22 @@ export function createCLI(options: CLIOptions = {}) {
         });
       },
     )
+    .command(
+      "purge",
+      "Completely wipe the specified LevelDB database",
```

Comment:
```suggestion
      "Completely wipe all files from the specified data directory",
```

Suggesting this to avoid a misunderstanding that it would only wipe a subset of files from the provided directory. As I understand, this totally nukes the entire directory.